examples/builder_derive_interop/README.md

```diff
@@ -0,0 +1,5 @@
+# Builder Derive Interop
```

Comment:
This is disconnected (not linked from) our current types of examples.  We have
- Derive tutorial
- Derive reference
- Use-case / application examples

I feel like this doesn't even need to be broken off as its own thing but could be part of one of those.  I see this being less about being a tutorial and more either part of the reference of a use case.  **Any thoughts on which this should be?**

If its part of the reference, I could see us having all of these `.md` files be directly in the reference, rather than a bunch of separate files.  If this is part of the use-case examples, I could see us having only one `.md` file and all of the files are subparts of it.

Reply:
I suppose it would be part of the derive reference.

Reply:
Ok